src/apps/Altinn.AccessManagement/src/Altinn.AccessMgmt.Persistence/Definitions/CompactRelationDefinition.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Altinn.AccessMgmt.Core.Models;
+using Altinn.AccessMgmt.Persistence.Core.Contracts;
+using Altinn.AccessMgmt.Persistence.Core.Definitions;
+
+namespace Altinn.AccessMgmt.Repo.Definitions;
+
+/// <inheritdoc/>
+public class CompactRelationDefinition : BaseDbDefinition<CompactRelation>, IDbDefinition
+{
+    /// <inheritdoc/>
+    public CompactRelationDefinition(DbDefinitionRegistry definitionRegistry) : base(definitionRegistry)
+    {
+    }
+
+    /// <inheritdoc/>
+    public void Define()
+    {
+        definitionRegistry.Define<CompactRelation>(def =>
+        {
+            def.SetVersion(1);
+            def.SetType(DbDefinitionType.View);
+
+            def.RegisterExtProperty<ExtCompactRelation>(t => t.FromId, t => t.From, functionName: "CompactEntity");
+            def.RegisterExtProperty<ExtCompactRelation>(t => t.RoleId, t => t.Role, functionName: "CompactRole");
+            def.RegisterExtProperty<ExtCompactRelation>(t => t.ViaId, t => t.Via, functionName: "CompactEntity", nullable: true);
+            def.RegisterExtProperty<ExtCompactRelation>(t => t.ViaRoleId, t => t.ViaRole, functionName: "CompactRole", nullable: true);
+            def.RegisterExtProperty<ExtCompactRelation>(t => t.ToId, t => t.To, functionName: "CompactEntity");
+            def.RegisterProperty(t => t.Reason);
+
+            def.SetQuery(BasicScript());
+
+            def.AddManualDependency<Entity>();
+            def.AddManualDependency<Role>();
+            def.AddManualDependency<RoleMap>();
+            def.AddManualDependency<Assignment>();
+            def.AddManualDependency<Delegation>();
+        });
+    }
+
+    private string BasicScript()
```

Comment:
Om denne spørringen i praksis er grunnlaget for hva som er gyldige connections, manger denne da å inkludere alle underenheter for From-enheter. 

Man får jo ut children på Compact entity men fordi de ellers ikke er i datagrunnlaget får man nå ikke slått opp underenheter som gyldige enheter man har fått connection fra (via hovedenhet).

Ref. bug: https://github.com/Altinn/altinn-authorization-tmp/issues/927

Reply:
Løses i egen task i etterkant av TT02 deploy.